ivy/array/__init__.py

```diff
@@ -368,6 +368,47 @@ def __lt__(self, other):
     @_native_wrapper
     def __le__(self, other):
         return ivy.less_equal(self._data, other)
+    """
+    Operator Examples
```

Comment:
The top part of the docstring should follow thae same structure as __add__ method which u will find in the same file if u scroll above. lmk if this is unclear.